source/variables/z-index.css

```diff
@@ -0,0 +1,12 @@
+/* 
+The z-indexes for the project. 
+z-index: var(--navIndex);
+*/
+:root {
+	--index--hidden: -100;
+	--index--base: 1;
+	--index--content: 10;
+	--index--nav: 50;
+	--index--tooltip: 1000;
```

Comment:
This has been a question I've had regarding Phoenix's `z-index` variables: why do we need to jump to `--index--tooltip: 1000` from `--index--nav: 50`? And then to `--index--modal: 8000`? Do they _really_ need to be that high? If we have these defined layers, can't we get away with, say, `100` and `200` or something? Or `75` and `100`? (That `8000` is just too close to the flashback-inducing `z-index: 999999` for my liking is maybe the real take away here.)